config/label_templates.yml

```diff
@@ -36,6 +36,6 @@ plate_lds_al_lib:
   :label_class: Labels::PlateLabelLdsAlLib
   :pmb_template: 'sqsc_96plate_label_template_code39'
```

Comment:
You removed the single quotes around the plate_cellaca_qc pmb_template, maybe be consistent and change others if not needed?

Reply:
Aggh, sorry, I was fighting with my linter/formatter at this point. It was auto-correcting everything on save, and I wanted to avoid touching more than necessary as I know there are a lot of people working on the same files at the moment. I'd love to get prettier enabled for limber, but want to wait until things are a bit quieter as otherwise I think someone my try to kill me when they merge in their work.